index.js

```diff
@@ -29,6 +30,13 @@ module.exports = function (name, version) {
 			if (version === 'latest') {
 				data = data.versions[data['dist-tags'].latest];
 			} else if (version) {
+				if (!data.versions[version]) {
+					var versions = Object.keys(data.versions);
+					version = semver.maxSatisfying(versions, version);
```

Comment:
@sindresorhus If you mean the lines 34 and 35
```
					var versions = Object.keys(data.versions);
					version = semver.maxSatisfying(versions, version);
```
then those cannot be put after line 40:
```
				data = data.versions[version];
```
because `data` would be undefined then.

The exception raised in line 37 could be eliminated, though - if you don't mind accessing `data.versions[undefined]` in line 40.

I made changes to tests and readme as you suggested in other comments but here I'm not sure what I'm supposed to do. Please clarify and I'll change it accordingly. Thanks.